pkgs/by-name/cr/crates-lsp/package.nix

```
@@ -0,0 +1,39 @@
+{
+  fetchFromGitHub,
+  lib,
+  nix-update-script,
+  rustPlatform,
```

```suggestion
  lib,
  stdenv,
  rustPlatform,
  fetchFromGitHub,
  nix-update-script,
```

Just adding `stdenv`. Feel free to keep the order as you prefer.